chainio/clients/elcontracts/writer.go

```diff
@@ -545,3 +550,49 @@ func (w *ChainWriter) RegisterForOperatorSets(
 
 	return receipt, nil
 }
+
+func (w *ChainWriter) RemovePermission(ctx context.Context, request RemovePermissionRequest) error {
```

Comment:
you need to send receipt back - so that you can populate transaction details on CLI. check the other write methods.